pkg/asset/cluster/tfvars.go

```diff
@@ -36,24 +40,76 @@ func (t *TerraformVariables) Dependencies() []asset.Asset {
 		&installconfig.InstallConfig{},
 		&bootstrap.Bootstrap{},
 		&machine.Master{},
+		&machines.Master{},
 	}
 }
 
 // Generate generates the terraform.tfvars file.
 func (t *TerraformVariables) Generate(parents asset.Parents) error {
 	installConfig := &installconfig.InstallConfig{}
-	bootstrap := &bootstrap.Bootstrap{}
-	master := &machine.Master{}
-	parents.Get(installConfig, bootstrap, master)
+	bootstrapIgnAsset := &bootstrap.Bootstrap{}
+	masterIgnAsset := &machine.Master{}
+	mastersAsset := &machines.Master{}
+	parents.Get(installConfig, bootstrapIgnAsset, masterIgnAsset, mastersAsset)
 
-	bootstrapIgn := string(bootstrap.Files()[0].Data)
+	bootstrapIgn := string(bootstrapIgnAsset.Files()[0].Data)
 
-	masterIgn := string(master.Files()[0].Data)
+	masterIgn := string(masterIgnAsset.Files()[0].Data)
 
-	data, err := tfvars.TFVars(installConfig.Config, bootstrapIgn, masterIgn)
+	var mastersList metav1.List
+	err := yaml.Unmarshal(mastersAsset.MachinesRaw, &mastersList)
+	if err != nil {
+		return errors.Wrap(err, "unmarshal master machine list")
+	}
+
+	masters := make([]*clusterapi.Machine, 0, len(mastersList.Items))
+	for _, item := range mastersList.Items {
+		var master clusterapi.Machine
+		err = yaml.Unmarshal(item.Raw, &master)
+		if err != nil {
+			return errors.Wrap(err, "unmarshal master machine item")
+		}
+		masters = append(masters, &master)
+	}
+
+	var vpcCIDR string
+	if installConfig.Config.Platform.AWS != nil {
+		vpcCIDR = installConfig.Config.Platform.AWS.VPCCIDRBlock
+	}
+
+	var bridge string
+	var image string
+	if installConfig.Config.Platform.Libvirt != nil {
+		vpcCIDR = installConfig.Config.Platform.Libvirt.Network.IPRange
```

Comment:
maybe name it same or keep them separate.

Reply:
> maybe name such that we can keep it same or keep them separate.

These are all the same idea.  And personally, I'd be fine promoting all of these to an `InstallConfig.Networking` property (alongside `ServiceCIDR` and `ClusterNetworks`) to make it more obvious that it's a cross-platform idea.  Maybe `nodeCIDR`?  `machineCIDR`?

Reply:
> InstallConfig.Networking

this is more about k8s cluster networking than cloud networking. we might need separation inside that.. and keeping the fields in cloud specific allows us to not aggregate the name...

but i keeping the cloud networking cidr in Networking seems fine too.

> nodeCIDR 
no node has a meaning in k8s